### PR TITLE
Add cli

### PR DESCRIPTION
**Pull request by syuhei176**

### CODE REVIEW

File: packages/cli/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    checkpointManager,
+    config
+  )
+}
+
+export default async function initialize() {
+  const lightClient = await instantiate()
+  console.log('start')
+  await lightClient.start()
+  console.log('started')
```
Comment (syuhei176) [inline packages/cli/src/index.ts]: `await lightClient.start()` never return. is it correct?

Comment (syuhei176) [reply packages/cli/src/index.ts]: maybe I should run lightClient.start() another frame or process.

File: packages/cli/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  return lightClient
+}
+
+/*
+token address is 000053c1edca8bd5c21c61d6f1fd091fa51d562aff1d
+console.log(
+  TezosMessageUtils.writeAddress('tz1TGu6TN5GSez2ndXXeDX6LgUDvLzPLqgYV')
+)
+*/
+console.log(
```
Comment (daiki44) [inline packages/cli/src/index.ts]: console.log?

File: packages/contract/src/DepositContract.ts [hunk 3]
```diff
@@ -35,11 +30,14 @@ export class DepositContract implements IDepositContract {
...
     this.connection = new ContractManager(wallet, address)
     this.eventWatcher = new EventWatcher({
+      tezosNodeEndpoint: wallet.tezosNodeEndpoint,
       conseilServerInfo: wallet.conseilServerInfo,
       kvs: eventDb,
       contractAddress: address.data
     })
-    this.tokenAddress = TezosMessageUtils.readAddress(address.data)
+    this.tokenAddress = TezosMessageUtils.readAddress(
+      '000053c1edca8bd5c21c61d6f1fd091fa51d562aff1d'
```
Comment (daiki44) [inline packages/contract/src/DepositContract.ts]: should we use variables?

File: packages/wallet/src/helpers/ContractManager.ts [hunk 4]
```diff
@@ -45,16 +46,22 @@ export class ContractManager {
...
       this.tzWallet.keyStore,
-      this.contractAddress.raw,
+      TezosMessageUtils.readAddress(this.contractAddress.data.substr(2)),
       amount,
       fee,
       derivationPath,
       storageLimit,
       gasLimit,
-      entrypoint,
+      undefined,
```
Comment (daiki44) [inline packages/wallet/src/helpers/ContractManager.ts]: undefined?

File: packages/wallet/src/helpers/ContractManager.ts [hunk 5]
```diff
@@ -46,9 +47,10 @@ export class ContractManager {
 
     const fee: number = await this.estimateFee()
     const result = await TezosNodeWriter.sendContractInvocationOperation(
-      this.tzWallet.conseilServerInfo.url,
+      'https://tezos-dev.cryptonomic-infra.tech',
```
[the remark this answers is not in the document]
Comment (syuhei176) [reply packages/wallet/src/helpers/ContractManager.ts]: I'm using `this.tzWallet.tezosNodeEndpoint` now.

File: packages/contract/src/DepositContract.ts [hunk 6]
```diff
@@ -96,11 +92,12 @@ export class DepositContract implements IDepositContract {
         }
       ]
     }
-    await this.connection.invokeContract(
-      amount.data,
+    const result = await this.connection.invokeContract(
+      0, //amount.data,
```
[the remark this answers is not in the document]
Comment (syuhei176) [reply packages/contract/src/DepositContract.ts]: fixed

File: packages/wallet/src/helpers/ContractManager.ts (hunk 4 above)
Comment (syuhei176) [reply packages/wallet/src/helpers/ContractManager.ts]: I got "no_such_entrypoint" by specifying entrypoint.

**Pull request merged**